Libplanet.Action.Tests/Common/DumbAction.cs

```diff
@@ -96,20 +109,32 @@ public IWorld Execute(IActionContext context)
 
             if (Transfer is { } transfer)
             {
-                world = world.TransferAsset(
-                    context,
-                    sender: transfer.From,
-                    recipient: transfer.To,
-                    value: FungibleAssetValue.FromRawValue(DumbCurrency, transfer.Amount),
-                    allowNegativeBalance: true);
+                world = (transfer.From, transfer.To) switch
+                {
+                    (Address from, Address to) => world.TransferAsset(
```

Comment:
I don't think it's crucial, but `from` is a query operator of `Linq`, so may be can cause confusion IMHO. Seems better to avoid if evitable.